data/obs-packaging/cc-oci-runtime/cc-oci-runtime.dsc-template

```diff
@@ -11,5 +11,5 @@ DEBTRANSFORM-RELEASE: 1
 
 Package: cc-oci-runtime
 Architecture: amd64
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}, qemu-lite, clear-containers-image, linux-container
+Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}, qemu-lite (>=@qemu_lite_pv@), clear-containers-image (>=@cc_image_pv@), linux-container (>=@linux_container_pv@)
```

Comment:
Is `pv` "packaging version"? I'd probably just make all these variables end in `_version` as that is clearer.